test/cli/integration_test.sh

```diff
@@ -0,0 +1,221 @@
+    exit 1
+fi
+echo "[OK]"
+
+echo -n "Checking pod resource limits: "
+if [[ -n $(kubectl get pod -l marblerun/marbletype=hello-world -o jsonpath='{.spec.containers[0].resources.limits}' | grep kubernetes.azure.com/sgx_epc_mem_in_MiB) ]];
+then
+    echo "[FAIL]"
+    exit 1
+fi
+echo "[OK]"
+
+
+#-------------test injection of sgx values---------------------------
+
+echo ""
+marblerun namespace add default
+
+if [[ -n $(kubectl describe namespace default | grep marblerun/inject | grep -v enabled) ]];
+then
+    echo "failed to inject correct values into namespace"
+    exit 1
+fi
+
+kubectl apply -f test-pod.yaml
+wait_for_resource "kubectl get pods -n default | grep default | grep -v Pending"
+
+if [[ -n $(kubectl logs -n marblerun -l app.kubernetes.io/name=marble-injector | tail -n 2 | grep -v "successful" | grep "omitting sgx injection") ]];
+then
+    echo "called the wrong webhook"
+    exit 1
+fi
+
+if [[ -n $(kubectl describe pod test-pod | grep FailedScheduling | grep -v "Insufficient kubernetes.azure.com/sgx_epc_mem_in_MiB") ]];
+then
+    echo "pod creating stuck in pending due to unrelated issue"
+    exit 1
+fi
+
+echo -n "Checking pod tolerations: "
+if [[ -n $(kubectl get pod test-pod -o jsonpath='{.spec.tolerations}' | grep -v "kubernetes.azure.com/sgx_epc_mem_in_MiB") ]];
+then
+    echo "[FAIL]"
+    exit 1
+fi
+echo "[OK]"
+
+echo -n "Checking pod resource limits: "
+if [[ -n $(kubectl get pod test-pod -o jsonpath='{.spec.containers[0].resources.limits}' | grep -v "kubernetes.azure.com/sgx_epc_mem_in_MiB") ]];
+then
+    echo "[FAIL]"
+    exit 1
+fi
+echo "[OK]"
+
+echo -n "Checking env variable EDG_COORDINATOR_ADDR: "
+if [[ -n $(kubectl get pod test-pod -o jsonpath='{.spec.containers[0].env[0]}' | grep -v coordinator-mesh-api.marblerun:25554) ]];
+then
+    echo "[FAIL]"
+    exit 1
+fi
+echo "[OK]"
+
+echo -n "Checking env variable EDG_MARBLE_TYPE: "
+if [[ -n $(kubectl get pod test-pod -o jsonpath='{.spec.containers[0].env[1]}' | grep -v test) ]];
+then
+    echo "[FAIL]"
+    exit 1
+fi
+echo "[OK]"
+
+echo -n "Checking env variable EDG_MARBLE_DNS_NAMES: "
+if [[ -n $(kubectl get pod test-pod -o jsonpath='{.spec.containers[0].env[2]}' | grep -v test,test.default,test.default.svc.cluster.local) ]];
+then
+    echo "[FAIL]"
+    exit 1
+fi
+echo "[OK]"
+
+echo -n "Checking env variable EDG_MARBLE_UUID_FILE: "
+if [[ -n $(kubectl get pod test-pod -o jsonpath='{.spec.containers[0].env[3]}' | grep -v "/test-uid/uuid-file") ]];
+then
+    echo "[FAIL]"
+    exit 1
+fi
+echo "[OK]"
+
+echo ""
+kubectl delete pod test-pod
+marblerun namespace remove default
+
+if [[ -n $(marblerun namespace list | grep default) ]];
+then
+    echo "failed to remove default namespace from the Marblerun mesh"
+    exit 1
+fi
+
+kubectl delete deployments hello-marble
+kubectl delete svc hello-marble
+marblerun uninstall
+kubectl delete namespace marblerun
+wait_for_resource "kubectl get namespaces | grep marblerun"
+
+pkill kubectl
```

Comment:
```suggestion
kill %1 # kill kubectl port-forward
```
If you just want to kill the background port-forward, this might be more targeted.